recipes/LibriSpeech/librispeech_prepare.py

```diff
@@ -284,6 +284,9 @@ def create_csv(
     """
     # Setting path for the csv file
     csv_file = os.path.join(save_folder, split + ".csv")
+    if os.path.exists(csv_file):
+        logger.info('Csv file %s already exists, not recreating.' % csv_file)
+        return
 
     # Preliminary prints
     msg = "Creating csv lists in  %s..." % (csv_file)
```

Comment:
We should push the symbolic link to "librispeech_prepare.py" under SSL, otherwise this script won't be found by default.

Reply:
done